app/src/main/java/org/glucosio/android/activity/PreferencesActivity.java

```diff
@@ -241,7 +240,7 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                 @Override
                 public boolean onPreferenceChange(Preference preference, Object newValue) {
                     if (newValue.toString().equals(getResources().getString(R.string.helloactivity_spinner_preferred_glucose_unit_1))) {
-                        updatedUser.setPreferred_unit("mg/dL");
+                        updatedUser.setPreferred_unit(DomainConstants.MG_D_L);
                     } else {
                         updatedUser.setPreferred_unit("mmol/L");
```

Comment:
Can add to `DomainConstants`.

Reply:
@emartynov I agree with @WBertan. Unit `"mmol/L` should be in `DomainConstants`

Reply:
Guys, there are much more thing like this. I'm creating new issue for this #386